ntbs-ui-tests/README.md

```diff
@@ -66,6 +66,8 @@ factor authentication within the tests). To do this you will need to:
   - The `AuthenticatedCookieHeader`to the value you copied from the "cookie" header.
   - The `Username` of the `ManuallyLoggedInUser`. This is the username of the user you used to log in above.
 
+Before running the test using the `phe-uat` environment, you must also update the value of `Category` in the file `test.ps1` from `NormalAuth` to `CookieOverride`.
```

Comment:
It's one thing to ask users to update a config file, but another to ask them to update an actual script. This feels too hacky to me.

I think a better way to do this would be to have a separate run script called something like `testWithCookieAuth.ps1` and then tell the user to run that script instead.

Given how simple the script is, I wouldn't mind if it's just a duplicate script with a different category. (If we had more time then you could commonise, but I don't think it's high enough priority to do that now).